wordpress/1.10/README.md

```diff
@@ -0,0 +1,151 @@
+
+## Install MySQL
+
+Next, we install and set up MySQL via the DC/OS CLI. For this, create a file called `config.json` and set the following properties:
+
+```json
+{
+  "service": {
+    "name": "mysql"
+  },
+  "database": {
+    "name": "wordpress",
+    "username": "wordpress",
+    "password": "password",
+    "root_password": "wordpress"
+  },
+  "networking": {
+    "port": 3306,
+    "host_mode": true
+  }
+}
+```
+
+Then, install MySQL:
+
+```bash
+$ dcos package install mysql --options=config.json 
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#community-services
+This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.
+
+```Advanced Installation options notes```
+
+storage / *persistence*: create local persistent volumes for internal storage files to survive across restarts or failures.
+
+storage / persistence / *external*: create external persistent volumes. This allows to use an external storage system such as Amazon EBS, OpenStack Cinder, EMC Isilon, EMC ScaleIO, EMC XtremIO, EMC VMAX and Google Compute Engine persistent storage. *NOTE*: To use external volumes with DC/OS, you MUST enable them during CLI or Advanced installation.
+
+storage / *host_volume*:  if persistence is not selected, this package can use a local volume in the host for storage, like a local directory or an NFS mount. The parameter *host_volume* controls the path in the host in which these volumes will be created, which MUST be the same on all nodes of the cluster.
+
+NOTE: If you didn't select persistence in the storage section, or provided a valid value for *host_volume* on installation,
+YOUR DATA WILL NOT BE SAVED IN ANY WAY.
+
+networking / *port*: This DC/OS service can be accessed from any other application through a NAMED VIP in the format *`service_name.marathon.l4lb.thisdcos.directory:port`*. Check status of the VIP in the *Network* tab of the DC/OS Dashboard (Enterprise DC/OS only).
+
+networking / *external_access*: create an entry in Marathon-LB for accessing the service from outside of the cluster
+
+networking / *external_access_port*: port to be used in Marathon-LB for accessing the service.
+Continue installing? [yes/no] yes
+Installing Marathon app for package [mysql] version [5.7.12-0.3]
+Service installed.
+
+Default login: `admin`/`password`. This username/password combination only applies if you haven't changed the defaults.
+```
+
+## Install WordPress
+
+To make WordPress accessible to the world, we need to use the fully qualified domain name of your DC/OS public agent. To install from the CLI, first create a `config.json` as below and replace the `virtual-host` property, i.e. `$PUBLIC_AGENT`, with your own (for example `http://ec2-52-51-80-141.eu-west-1.compute.amazonaws.com`):
+
+```json
+{
+  "database": {
+    "host": "mysql.marathon.mesos:3306",
```

Comment:
Could we change this to using a VIP? That could also remove the need for a fix port...